Database/Corrections/sodQuestFixes.lua

```diff
@@ -3411,6 +3411,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425344,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90230] = {
+            [questKeys.name] = "Focused Rage",
+            [questKeys.startedBy] = {{2554,2555,2556,218032},{450013}},
```

Comment:
We shouldn't show the NPCs. The regular trolls will flood the map and the other one has nearly the same location as the gong.

```suggestion
            [questKeys.startedBy] = {nil,{450013}},
```